src/Monolog/Logger.php

```diff
@@ -137,12 +160,15 @@ class Logger implements LoggerInterface
      * @param string             $name       The logging channel
      * @param HandlerInterface[] $handlers   Optional stack of handlers, the first one in the array is called first, etc.
      * @param callable[]         $processors Optional array of processors
+     * @param string             $level Minimal level to log
```

Comment:
I think the first letter of comment to the parameter should be adjusted to the same level as others.

Reply:
Fixed